src/Compilers/Core/Portable/Diagnostic/DiagnosticDescriptor.cs

```diff
@@ -40,9 +40,9 @@ public sealed class DiagnosticDescriptor : IEquatable<DiagnosticDescriptor>
         public LocalizableString MessageFormat { get; }
 
         /// <summary>
-        /// The category of the diagnostic (like Design, Naming etc.)
+        /// The localizable category of the diagnostic (like Design, Naming etc.)
         /// </summary>
-        public string Category { get; }
+        public LocalizableString Category { get; }
```

Comment:
Is it weird that Category returns a `LocalizableString` but `GetMessageFormat` returns an already-localized `string`?